index.html

```diff
@@ -100,6 +97,12 @@ <h4 class="post-title upper">عروسک های قلاب بافی</h4>
                 </div>
             </div>
         </div>
+        <footer>
+            <div class="footer">استفاده از مطالب فروشگاه قلاب بانو فقط برای مقاصد غیرتجاری و با ذکر منبع بلامانع است.
```

Comment:
vaghti ye tag footer darin dg asalan dorost nist be ye tag dg ye class ba nam **footer** bedin. chon adam vaghti in footer ro mibine montazare tag footer e.  lotfan namesho taghyir bedin . maslan mitoone footerContent bashe ya harchi